osdep/macOS_mpv_helper.swift

```diff
@@ -101,8 +102,9 @@ class MPVHelper: NSObject {
         if mpvGLCBContext != nil {
             var i: GLint = 0
             glGetIntegerv(GLenum(GL_DRAW_FRAMEBUFFER_BINDING), &i)
+            fbo = i != 0 ? i : fbo
```

Comment:
This might be overblown length-wise, but IMHO something like this definitely requires a comment.

```swift
            // CAOpenGLLayer has ownership of FBO zero yet can return it to us,
            // so only utilize a newly received FBO ID if it is nonzero.
            //
            // Otherwise fall back to previously utilized FBO. Initial fallback
            // default is FBO 1.
            fbo = i != 0 ? i : fbo
```

Reply:
k done. i just used your comment since it's fine.